### DISCUSSION TITLE
E22-900M30S SX1262 Output TX Power Configuration

### DISCUSSION THREAD

**Discussion by ghost** [jgromes/RadioLib#790] [Q&A]

**Comment by jgromes** [comment]
I don't have the datasheet on hand, but a couple of points to consider:

* PA gain is frequency dependent
* What is your power source? Can it handle the PA load?

EDIT: Also, not really an issue since this is definitely out of scope of the library, so moving to discussion

**Comment by ghost** [comment]


**Reply by jgromes** [reply]
This seems like a question to the module manufacturer. RadioLib can only configure the SX126x, not the RF components beyond it.
